### PR TITLE
CMake: Move version value managing to top of CMakeList.txt

### PR DESCRIPTION
**Pull request by Yeongjin-Jeong**
Using the cmake environment variable as the version value simplifies versioning the pkg-config file or API header.

And the version value of the pkg-config file and API header has not changed in the latest release tag, so it was bumped up version to 1.4.0.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -31,8 +31,14 @@ if(NOT DEFINED CMAKE_INSTALL_PREFIX OR CMAKE_INSTALL_PREFIX STREQUAL "")
 endif()
 message(STATUS "CMAKE_INSTALL_PREFIX = ${CMAKE_INSTALL_PREFIX}")
 
+set(SVT_VERSION_MAJOR           "1")
+set(SVT_VERSION_MINOR           "4")
+set(SVT_VERSION_PATCHLEVEL      "0")
```
Comment (1480c1) [inline CMakeLists.txt]:
```suggestion
set(SVT_HEVC_VERSION_PATCHLEVEL      "0")
```

Change to SVT_HEVC to separate out from the other SVT encoders?

Comment (Yeongjin-Jeong) [reply CMakeLists.txt]: Does that mean only cmake variables ? If that is right, I applied it :)

Comment (1480c1) [reply CMakeLists.txt]: Yes

**Comment by tianjunwork** [comment]
Hi @Yeongjin-Jeong, thanks for your contribution. Thanks @1480c1 for helping code review.

**Pull request merged**